components/collection/drop/MintButton.vue

```diff
@@ -50,11 +55,20 @@ const isHolderAndEligible = computed(
     holderOfCollection.value.hasAvailable,
 )
 
+watch(drop, async () => {
+  const tokenPrice = await getApproximatePriceOf(chainSymbol.value)
+  const tokenAmount = calculateBalanceUsdValue(
+    Number(drop.value.price),
+    decimals.value,
+  )
+  priceUsd.value = `${Number(tokenAmount * tokenPrice)}`
```

Comment:
Why convert it to string?
```suggestion
  priceUsd.value = tokenAmount * tokenPrice
```

Reply:
no real reason.. fixed